docs/user/tutorials/01-30-20-aws-vpc-peering.md

```diff
@@ -1,24 +1,23 @@
-# Create VPC Peering in AWS
+# Create Virtual Private Cloud Peering in Amazon Web Services
 
-This tutorial explains how to create a VPC peering connection between a remote VPC network and Kyma in AWS. Follow the 
-steps from this tutorial to create a new VPC network, and VM, and assign required permissions to the provided Kyma account and role in your AWS account. If you want to
-use the existing resources instead of creating new ones, adjust variable names accordingly and skip the steps that 
-create those resources.
+This tutorial explains how to create a Virtual Private Cloud (VPC) peering connection between a remote VPC network and Kyma in Amazon Web Services (AWS). Learn how to create a new VPC network, and a virtual machine (VM), and assign required permissions to the provided Kyma account and role in your AWS account. If you want to use the existing resources instead of creating new ones, adjust variable names accordingly and skip the steps that create those resources.
```

Comment:
```suggestion
This tutorial explains how to create a Virtual Private Cloud (VPC) peering connection between a remote VPC network and Kyma in Amazon Web Services (AWS). Learn how to create a new VPC network, and a virtual machine (VM), and assign required permissions to the provided Kyma account and role in your AWS account. If you want to use the existing resources instead of creating new ones, adjust variable names accordingly and skip the steps that create or delete those resources.
```

Should we put some message in the clean-up section as a heads up to the user not to delete existing resources.

Reply:
Or we can simply live out the whole part that suggest the user to use existing resources. Perhaps it is better this way. What do you think?